第三回課題提出

Yuto0007 opened the pull request:
第3回課題を提出いたします。
ご確認よろしくお願いいたします。

Commit:
add lecture03.md

Yuto0007 committed:
Create an image directory

Commit:
lecture03.mdの下書き

Commit:
Merge branch 'lecture03' of github.com:yuto-kabaya/RaiseTech into lecture03
リモートからもローカルからも変更を加えてしまったので、修正するため。

Commit:
lecture03.mdの修正

Commit:
lecture03.mdの修正

Yuto0007 committed:
Delete lecture03/image directory

Yuto0007 committed:
Create lecture03 directory

Yuto0007 committed:
Create empty

Yuto0007 committed:
Delete lecture03/empty

Commit:
lecture03.mdに画像埋め込み

Yuto0007 committed:
画像をimageディレクトリに追加

dai-fuji reviewed (commented):
内容は良さそうですね！
エビデンスの画像もリポジトリにあげていただいていますが、これらをマークダウンの適切な場所に埋め込んでいただけますか？
マークダウンファイルを閲覧するだけで、エビデンスも一緒に確認できる状態を目指してください！

Yuto0007 committed:
Delete lecture03.md

lecture03ブランチにlecture03.mdが２種類（画像埋め込み前と画像埋め込み後）あったため、lecture03ディレクトリ内にあるlecture03.mdを残し、それ以外は削除しました。

yuto-kabaya commented:
コメントありがとうございます！
lecture03ブランチにlecture03.mdが２種類（画像埋め込み前と画像埋め込み後）あったため、lecture03ディレクトリ内にあるlecture03.mdを残し、それ以外は削除しました。
ご確認よろしくお願いいたします。

dai-fuji commented on lecture03/image/empty:
こちら必要なファイルでしょうか？？

yuto-kabaya replied:
emptyファイルは不要なファイルです。
imageディレクトリ作成時に作成して削除し忘れていたので、ただ今削除しました。
ご確認よろしくお願いいたします。

dai-fuji reviewed (commented):
一点コメントしております！

Yuto0007 committed:
Delete lecture03/image/empty

不要なファイルのため削除しました。

dai-fuji reviewed (approved):
OKです🙆‍♂️
